cda-pentaho-base/resources/previewer/languages/Messages_ja.properties

```diff
@@ -1,4 +1,3 @@
-label.refresh=\u66f4\u65b0
-label.selectDataAccessID=\u30c7\u30fc\u30bf \u30a2\u30af\u30bb\u30b9 ID \u3092\u9078\u629e\u3057\u3066\u304f\u3060\u3055\u3044\u3002
-label.doExport=.xls \u30d5\u30a1\u30a4\u30eb\u306b\u30a8\u30af\u30b9\u30dd\u30fc\u30c8
+label.refresh=\u30ea\u30d5\u30ec\u30c3\u30b7\u30e5
+label.selectDataAccessID=\u30c7\u30fc\u30bf \u30a2\u30af\u30bb\u30b9 IDlabel.export=Export as .xls \u3092\u9078\u629e\u3057\u3066\u4e0b\u3055\u3044\u3002
```

Comment:
As in the 6.0 pull request, label.export should be at the beginning of a new line.